particula/aerosol_dynamics/parcel.py

```diff
@@ -1,45 +1,35 @@
 """
 Class for creating a air parcel to put particles and gases in.
 """
-
-from particula.aerosol_dynamics import environment, particle
+import numpy as np
+from particula.aerosol_dynamics import particle, u
 
 
 class Parcel:
-    """Sets the class for creating a parcel.
+    """
+    Sets the class for creating a parcel.
```

Comment:
We generally want to be more descriptive and concise. Like it would appear in these `help` calls.

Reply:
better?

Reply:
I don't see any changes yet...